CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Full documentation for hipCUB is available at [https://rocm.docs.amd.com/projects/hipCUB/en/latest/](https://rocm.docs.amd.com/projects/hipCUB/en/latest/).
 
+## (Unreleased) hipCUB-x.x.x for ROCm 6.x.x
```

Comment:
We don't know what the next ROCm release/version will be, so I'm okay with leaving these placeholders in the changelog for now, but we'll need to fix this once we know what the next release version is.